README.md

```diff
@@ -8,27 +8,31 @@ A collection of XML schema's used by EASY. These schema's are published at
 Maintain sword-v1-packaging
 ---------------------------
 
-The Sword-v1 [service document] (ignore login) refers to the [packaging document]
-located next to the ddm [examples]. The `<article id='mapping'>` in the packaging document
+The Sword-v1 [service document] \(ignore login) refers to the published [packaging document], its source is
```

Comment:
that link works fine now! Nice work. Didn't know you could escape things in Markdown...

One issue with this link though (didn't realize this before): it leads to something that is behind an authentication.

1. is that correct?
2. Should we refer to this '*private*' link on this repository, given that it is a **public** repository?

Would like to hear @janvanmansum on this.